File: beginner/1013/the_greatest.c
#include <stdio.h>

int	abs_(int n)
{
	if (n < 0)
		return (n * -1);
	else
		return (n);
}

int	greatest(int a, int b)
{
	int gt;

	gt = (a + b + abs_(a - b));
	return (gt / 2);
}

int	main(void)
{
	int num_a;
	int	num_b;
	int	num_c;
	int	greater;

	scanf("%d %d %d", &num_a, &num_b, &num_c);
	greater = greatest(num_a, num_b);	
	greater = greatest(greater, num_c);
	printf("%d eh o maior\n", greater);
}

File: beginner/1013/TheGreatest.java
import java.io.IOException;
import java.util.Scanner;
import java.lang.Math;

public class TheGreatest {

	public static void main(String[] args) throws IOException {
		Scanner reader = new Scanner(System.in);
		int a, b, c, greatest;
		a = reader.nextInt();
		b = reader.nextInt();
		c = reader.nextInt();
		reader.close();
		greatest = getGreatest(a, b);
		greatest = getGreatest(greatest, c);
		System.out.println(greatest + " eh o maior");
	}

	public static int getGreatest(int a, int b) {
		return (a + b + Math.abs(a - b)) / 2;
	}
}
